### PR TITLE
Feature/get and post style

### PR DESCRIPTION
**Pull request by KCWill**
Add axios get and post functionality with the backend. Display computed style on style preferences page. #24 #23

### CODE REVIEW

File: code/web/src/modules/user/api/actions.js [hunk 1]
```diff
@@ -103,11 +104,42 @@ export function logoutUnsetUserLocalStorageAndCookie() {
   // Remove token
   window.localStorage.removeItem('token')
   window.localStorage.removeItem('user')
-
+  
   // Remove cookie
   cookie.remove('auth')
 }
 
+export function fetchStylePreference(userId) {
```
Comment (jrsewell400) [inline code/web/src/modules/user/api/actions.js:112]: It was really interesting to walk through writing these fetchStyle and setStyle functions.  I feel as though I have a decent grasp of their purpose and what is happening within each.  Thanks for that, Kim and Kyle!

File: code/web/src/modules/user/api/state.js [hunk 2]
```diff
@@ -84,6 +84,11 @@ export const surveyReducer = (state = surveyInitialState, action) => {
         ...state,
         determinedStyle: action.payload
       } 
+    case 'SAVED_SURVEY_RESULTS':
+      return {
+        ...state,
```
Comment (jrsewell400) [inline code/web/src/modules/user/api/state.js:89]: I'm happy to know how state works, seems to be a very powerful tool for front end

File: code/api/src/modules/user/mutation.test.js [hunk 3]
```diff
@@ -35,7 +35,6 @@ describe("user mutations", () => {
     .post('/')
     .send({ query: 'mutation{ userUpdate(id: 1, stylePreferences: \"classic\"){ stylePreferences }}' })
     .expect(200)
-    console.log(response.body)
```
Comment (jrsewell400) [inline code/api/src/modules/user/mutation.test.js:38]: Thanks for removing my leftover console log, Kyle!

**Comment by kimalajoy** [comment]
Great job executing this functionality. Looks great.

**Pull request merged**